src/api/queues.js

```diff
@@ -113,13 +113,25 @@ router.get(
           },
           required: false,
           include: [User],
+          attributes: ['startTime', 'endTime'],
```

Comment:
Can you clarify why the changes in this file were necessary?

Reply:
I can't find the exact change from v5, but without `attributes`, `sequelize` will throw an error stating `include.attributes.map is not a function`.